lib/components/user/monitored-trip/trip-summary-pane.tsx

```diff
@@ -40,9 +41,9 @@ const LocationDetails = styled.div`
 
   @media (max-width: 768px) {
     align-items: center;
-    border-bottom: 1px solid #ddd;
+    border-bottom: 1px solid ${grey[100]};
     border-right: 0;
-    border-top: 1px solid #ddd;
+    border-top: 1px solid ${grey[100]};
```

Comment:
What if we created a BORDER_COLOR variable for this file? I am hesitant to create one for all instances of grey[100] because we use it for a lot of different purposes.

Reply:
> What if we created a `BORDER_COLOR` variable for this file? I am hesitant to create one for all instances of `grey[100]` because we use it for a lot of different purposes.

Sounds good.

Reply:
`grey[100]` is probably a light grey border, as opposed to `grey[800]` used in `printed-styles`.

Reply:
The `printed-styles` border is actually inside of another light grey container, `grey[800]` (`#535353`) seems like it's going to look better than `grey[100]` in this case

<img width="434" alt="image" src="https://github.com/opentripplanner/otp-react-redux/assets/115499534/4fac9b89-2dec-494b-9e0c-4e70d2483319">